src/gdcdictionary/schemas/_terms.yaml

```diff
@@ -6879,13 +6881,15 @@ wilms_tumor_histologic_subtype:
 workflow_type:
   common:
     description: >-
-      Generic name for the workflow used to analyze a data set.
+      The name of the workflow used to process and/or analyze a data set.
     termDef:
       term:
-      source:
-      cde_id:
-      cde_version:
-      term_url:
+      source: NCIt
+      cde_id: C181725
```

Comment:
Is there a resource we can leverage to confirm the cde_id?

Reply:
Yes, the NCIt browser: https://ncit.nci.nih.gov/ncitbrowser/ 

The site does go down from time to time FYI, I was having trouble with it today.